crates/sovereign-sdk/rollup-interface/src/state_machine/da.rs

```diff
@@ -39,11 +48,13 @@ impl core::cmp::Ord for SequencerCommitment {
 /// Data written to DA can only be one of these two types
```

Comment:
`TODO: rename to da service request smth smth` can be removed

Reply:
https://github.com/chainwayxyz/citrea/pull/1630/commits/267b23b939ff0b6ed426ed0aad118dd081eae5a0